web/ci.sh

```diff
@@ -83,4 +90,121 @@ if builder_start_action validate-size; then
   ./src/tools/building/check-build-size.sh $FLAGS
 
   builder_finish_action success validate-size
+fi
+
+if builder_start_action publish-s.keyman; then
+  # First phase: make sure the s.keyman.com repo is locally-available and up to date.
+  pushd "$S_KEYMAN_COM"
+  if builder_has_option --password; then
+    git pull https://keyman-server:$PASSWORD@github.com/keymanapp/s.keyman.com.git master
+  else
+    # For testing on a local development machine / a machine with the repo already loaded.
+    git checkout master
+    git pull
+  fi
+  popd
+
+  # Second phase:  copy the artifacts over
+
+  # The main build products are expected to reside at the root of this folder.
+  BASE_PUBLISH_FOLDER="$S_KEYMAN_COM/kmw/engine/$BUILD_NUMBER"
+  mkdir "$BASE_PUBLISH_FOLDER"
+
+  cp -Rf build/app/web/release/* "$BASE_PUBLISH_FOLDER"
+  cp -Rf build/app/ui/release/* "$BASE_PUBLISH_FOLDER"
+
+  # Third phase: tweak the sourcemaps
+  # We can use an alt-mode of Web's sourcemap-root tool for this.
+  for sourcemap in "$BASE_PUBLISH_FOLDER/"*.map; do
+    node build/tools/building/sourcemap-root/index.mjs null "$sourcemap" --sourceRoot "https://s.keyman.com/kmw/engine/$BUILD_NUMBER/src"
+  done
+
+  # Final phase:  build the PR and push it.
+  cd "$S_KEYMAN_COM"
+  if builder_has_option --password; then
+    git config user.name "Keyman Build Server"
+    git config user.email "keyman-server@users.noreply.github.com"
+  fi
+  git add "kmw/engine/$BUILD_NUMBER"
+  if builder_has_option --password; then
+    git commit -m "KeymanWeb release $BUILD_NUMBER (automatic)"
+    git push https://keyman-server:$PASSWORD@github.com/keymanapp/s.keyman.com.git master
+  fi
```

Comment:
Per discussion in alternative channels, we'll remove this section and have that continue to be handled by the same mechanisms currently used.  We'll continue to prep the files here... we just won't build the PRs here.